cmds/jrsonnet/src/main.rs

```diff
@@ -182,7 +182,7 @@ fn main_real(opts: Opts) -> Result<(), Error> {
 		let input_str = std::str::from_utf8(&input)?;
 		s.evaluate_snippet("<stdin>".to_owned(), input_str)?
 	} else {
-		s.import(&input)?
+		s.import(input.as_str())?
```

Comment:
Not entirely correct, as now imports also follow JPATH handling, which isn't behavior used by official jsonnet. JPATH handling should be disabled here.

To be split into resolve/import_resolved calls, with JPATH being applied in between those calls?